src/java/org/apache/cassandra/db/compaction/TimeWindowCompactionStrategyOptions.java

```diff
@@ -45,7 +46,7 @@
     public static final String EXPIRED_SSTABLE_CHECK_FREQUENCY_SECONDS_KEY = "expired_sstable_check_frequency_seconds";
     public static final String UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION_KEY = "unsafe_aggressive_sstable_expiration";
 
-    static final String UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION_PROPERTY = Config.PROPERTY_PREFIX + "allow_unsafe_aggressive_sstable_expiration";
+    static final boolean UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION_ENABLED = ALLOW_UNSAFE_AGGRESSIVE_SSTABLE_EXPIRATION.getBoolean();
```

Comment:
Should not this go away? If you check the logic, that property is resolved in runtime. What if we have dynamic changing of properties as we plan it? Then we could not tweak this in runtime because it is set in stone, no?

Reply:
Ok, so I've checked and the constant is used for `validateOptions` which is only related to tests, so you're probably right and we can get rid of it. 
However, my thought here is that removing such constants may trigger unnecessary questions during the patch review - accessing the system property is expensive and this value is "cached" in a class field. If we remove it, the question is "why?", so in general we are not doing anything wrong by leaving it "as is". 

So, if you still think we can do it - I'll do :-)